EFCore.NamingConventions.Test/RewriterTestBase.cs

```diff
@@ -8,10 +9,15 @@ public abstract class RewriterTestBase
     {
         public class TestContext : DbContext
         {
-            readonly Func<DbContextOptionsBuilder, DbContextOptionsBuilder> _useNamingConvention;
-            
-            public TestContext(Func<DbContextOptionsBuilder, DbContextOptionsBuilder> useNamingConvention)
-                => _useNamingConvention = useNamingConvention;
+            readonly Func<DbContextOptionsBuilder, CultureInfo, DbContextOptionsBuilder> _useNamingConvention;
+            readonly CultureInfo _culture;
+
+
+            public TestContext(Func<DbContextOptionsBuilder, CultureInfo, DbContextOptionsBuilder> useNamingConvention, CultureInfo culture = null)
```

Comment:
u mean should i revert back to original function? then how can i pass Culture with   
Func<DbContextOptionsBuilder, DbContextOptionsBuilder> ?

Reply:
This Func is configures the context in any way necessary. Before your PR, the Func simply is UseSnakeCaseNamingConvention, but it can also be a call to UseSnakeCaseNamingConvention with a parameter.

In other words, instead of:

```c#
new TestContext(NamingConventionsExtensions.UseSnakeCaseNamingConvention)
```

it should simply be:

```c#
new TestContext(() => NamingConventionsExtensions.UseSnakeCaseNamingConvention(CultureInfo.Whatever))
```